app/src/components/AddProposalPanel.js

```diff
@@ -4,33 +4,35 @@ import styled from 'styled-components'
 
 const AddProposalPanel = ({ onSubmit }) => {
   const [title, setTitle] = useState('')
-  const [description, setDescription] = useState('Lorem ipsum...')
+  const [link, setLink] = useState('')
   const [amount, setAmount] = useState(0)
-  const [beneficiary, setBeneficiary] = useState('')
+  const [recipient, setRecipient] = useState('')
   const disabled = false // TODO Disable when empty or invalid fields
-  const onFormSubmit = event => {
-    event.preventDefault()
-    onSubmit({ title, description, amount, beneficiary })
-  }
   return (
-    <form onSubmit={onFormSubmit}>
-      <Field label="Title">
+    <Form onSubmit={() => onSubmit({ title, link, amount, recipient })}>
```

Comment:
Please, do not remove the onFormSubmit function because an `event.preventDefault()` should be called to not refresh the page when the form is sent. It can also be used later for checking the validity of fields.

Reply:
I added an `e.preventDefault()` but on the onClick method. 

I don't think that having the onSubmit inside the render is a good pratice. See if it's fine like this, now

Reply:
Both have the same impact on performance (an anonymous function must be created on every render), despite the first version was a bit more readable because it's not inside the JSX.